syncer/syncer.go

```diff
@@ -1804,6 +1804,13 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 			return terror.ErrSyncerGetEvent.Generate(err)
 		}
 
+		failpoint.Inject("OnlySomeTypeEventCanPass", func(val failpoint.Value) {
```

Comment:
we should only skip "HeartbeatEvent". IMO we should revert this change of failpoint